package.json

```diff
@@ -13,10 +13,10 @@
     "test": "./test"
   },
   "dependencies": {
+    "@elastic/elasticsearch": "^7.14.1",
```

Comment:
I saw your comment: https://github.com/Karql/elastalert2-server/issues/12#issuecomment-914427326

Like you said elastalert2 for now only support es7.
https://github.com/jertel/elastalert2/blob/2.2.1/requirements.txt#L7

So for me it is also ok to support only latest version without if|else like here: https://github.com/johnsusek/elastalert-server/blob/master/src/common/elasticsearch_client.js

Getting client in this approche is also simpler - don't need to call elasitc with second lib to check version.

Btw. this client for now is used only for very simple cases so imo it should works with older elastic versions too.